### PR TITLE
Fix coverage glob

### PR DESCRIPTION
**Pull request by FrederikBolding**
**Description**

Fix glob that determined which files we collect coverage for

- FIXED:
  - Fix glob that determined which files we collect coverage for

**Checklist**

- [x] Tests are included if applicable
- [x] Any added code is fully documented

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -2,16 +2,16 @@ module.exports = {
   collectCoverage: true,
   // Ensures that we collect coverage from all source files, not just tested
   // ones.
-  collectCoverageFrom: ['./src/**.ts'],
+  collectCoverageFrom: ['**/src/**/*.ts'],
```
Comment (Gudahtt) [inline jest.config.js:5]: Nit: We only have one `src` directory, so this would suffice:
```suggestion
  collectCoverageFrom: ['*/src/**/*.ts'],
```

**Review by Gudahtt — DISMISSED** [review]
LGTM!

File: jest.config.js (hunk 1 above)
Comment (FrederikBolding) [reply jest.config.js:5]: That doesn't work according to: https://globster.xyz/ 🤔

Comment (Gudahtt) [reply jest.config.js:5]: Sorry, typo, I meant this:
```suggestion
  collectCoverageFrom: ['./src/**/*.ts'],
```

The added asterisk is the fix here I think

Comment (FrederikBolding) [reply jest.config.js:5]: Weirdly that doesn't seem to be enough 🤔

Comment (FrederikBolding) [reply jest.config.js:5]: Oh, nevermind that must be an issue with globster.

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**